.github/workflows/release.yml

```diff
@@ -28,12 +28,15 @@ jobs:
           fetch-depth: 0
           ref: "master"
 
-      - name: Set up Go
+      - name: Set up Go ${{ matrix.go-version }}
         uses: actions/setup-go@93397bea11091df50f3d7e59dc26a7711a8bcfbe
         with:
           go-version: ${{ matrix.go-version }}
           cache-dependency-path: vmware-event-router/go.sum
 
+      - name: Setup ko
+        uses: actions/setup-ko@v0.6
```

Comment:
use correct repo and SHA (IIRC required by OSPO security)

`ko-build/setup-ko@@ace48d793556083a76f1e3e6068850c1f4a369aa`